Nos/Views/Profile/Edit/CreateUsernameWizard/AlreadyHaveANIP05View.swift

```diff
@@ -124,31 +124,9 @@ struct AlreadyHaveANIP05View: View {
     }
 }
 
-fileprivate class UsernameObserver: ObservableObject {
-
-    @Published
-    var debouncedText = ""
-
-    @Published
-    var text = ""
-
-    private var subscriptions = Set<AnyCancellable>()
-
-    init() {
-        $text
-            .removeDuplicates()
-            .filter { $0.count >= 3 }
-            .debounce(for: .milliseconds(200), scheduler: RunLoop.main)
-            .sink { [weak self] value in
-                self?.debouncedText = value
-            }
-            .store(in: &subscriptions)
-    }
-}
-
 fileprivate struct UsernameTextField: View {
 
-    @StateObject var usernameObserver: UsernameObserver
+    @State var usernameObserver: TextDebouncer
```

Comment:
```suggestion
    var usernameObserver: TextDebouncer
```

Same reasoning as above for `SearchController`.